storage/txcache/txListForSender.go

```diff
@@ -101,10 +101,10 @@ func (listForSender *txListForSender) isCapacityExceeded() bool {
 	return tooManyBytes || tooManyTxs
 }
 
-func (listForSender *txListForSender) onAddedTransaction(tx *WrappedTransaction) {
+func (listForSender *txListForSender) onAddedTransaction(tx *WrappedTransaction, gasHandler TxGasHandler, txFeeHelper feeHelper) {
 	listForSender.totalBytes.Add(tx.Size)
 	listForSender.totalGas.Add(int64(estimateTxGas(tx)))
-	listForSender.totalFee.Add(int64(estimateTxFee(tx)))
+	listForSender.totalFee.Add(int64(estimateTxFeeScore(tx, gasHandler, txFeeHelper)))
```

Comment:
Not sure, can we use the same `TxFeeScoreNormalized`, as for `onRemovedListElement`?

Reply:
this is where TxFeeScoreNormalized is first computed and cached, upon add.